doc-Managing_Providers/topics/Storage_Managers.adoc

```diff
@@ -1,51 +1,49 @@
 [[storage_managers]]
 = Storage Managers
 
-In {product-title}, a storage manager is a service providing storage resources that you can manage from a {product-title} appliance. This chapter describes the different types of storage managers used by {product-title}, and how to manage them.
+In {product-title}, a storage manager is a service providing storage resources that you can manage from a {product-title} appliance. This chapter describes the different types of storage managers used by {product-title}, and how they are added to {product-title}.
 
-include::common/openstack/storage-manager-available.adoc[]
+There are three types of storage managers currently available to {product-title}: 
 
-:leveloffset: 2
-= OpenStack Block Storage Managers
-
-The OpenStack Block Storage service (`openstack-cinder`) provides and manages persistent block storage resources that OpenStack infrastructure instances can consume.
-
-To use OpenStack Block Storage as a storage manager, you must first add an OpenStack cloud provider to your {product-title} appliance and enable events. The Block Storage service will be automatically discovered by CloudForms and added to the *Storage Managers* list in {product-title}. See xref:adding_openstack_cloud_providers[] for instructions on adding a cloud provider and enabling events.
+* Amazon Elastic Block Store
+* OpenStack Block Storage (`openstack-cinder`) 
+* OpenStack Object Storage (`openstack-swift`)
 
+:leveloffset: 2
+= Amazon Elastic Block Store Managers
 
+The Amazon Elastic Block Store service provides and manages persistent block storage resources that Amazon EC2 instances can consume.
 
-:leveloffset: 3
-include::Creating_Cloud_Volumes_on_Cinder.adoc[]
-
-:leveloffset: 3
-include::Creating_a_Cinder_Backup.adoc[]
-
-:leveloffset: 3
-include::Restoring_from_a_Cinder_Backup.adoc[]
-
-:leveloffset: 3
-include::Creating_a_Cinder_Volume_Snapshot.adoc[]
+To use the Amazon Elastic Block Store service as a storage manager, you must first add an Amazon EC2 cloud provider to your {product-title} appliance. The Amazon Elastic Block Store service is automatically discovered by {product-title}, and added to the *Storage Managers* list. See xref:adding-amazong-ec2-providers[] for instructions on adding an Amazon EC2 cloud provider.
```

Comment:
All good - you are completely correct! Fixed in dfdc93eddec353d1e29fd988a0e4663d1ea11070.